apps/web/src/models/inventory/cloud-service/index.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+import type { Query } from '@cloudforet/core-lib/space-connector/type';
+
+
+
+
+// excel Export
+export const QueryType = {
+    ANALYZE: 'ANALYZE',
+    SEARCH: 'SEARCH',
+} as const;
+
+export type QueryType = typeof QueryType[keyof typeof QueryType];
+
+
+type SearchQueryField = {
+    key: string;
+    name: string;
+    reference?: any;
+    [key: string]: any;
+} | string;
+
+interface SearchQuery {
+    filter?: Query['filter'];
+    filter_or?: Query['filter_or'];
+    sort?: Query['sort'];
+    fields?: Array<SearchQueryField>;
+    keyword?: string;
+}
+
+interface AnalyzeQuery {
+    filter?: Query['filter'];
+    filter_or?: Query['filter_or'];
+    sort?: Query['sort'];
+    fields?: any;
+    labels?: string[];
+    group_by?: string[];
+    keyword?: string;
+}
+
+export interface ExportOption {
+    name: string; // name of the sheet
+    query_type: QueryType;
+    search_query?: SearchQuery;
+    analyze_query?: AnalyzeQuery;
+}
+
+export interface CloudServiceExportParameter {
+    options: ExportOption[]
+    timezone?: string;
+    file_format?: 'Excel' | 'CSV'; // default: Excel
+}
```

Comment:
suggestion: Except for this interface, whole other type declaration is about 'data-export' feature. 
Also, `CloudServiceExportParameter` spec seems very common. 
What about to change this file to `apps/web/src/models/data-export.ts`?
And then change `CloudServiceExportParameter` to more common name?

I heard that data export feature will be added to other services when it is needed. 
And I think this gonna be the same spec for all export api for other services.
So it would be more efficient to declare it as common feature once.